src/test/java/api/support/ApiRoot.java

```diff
@@ -10,6 +10,10 @@ public static String inventory() {
     return String.format("%s/inventory", ApiTestSuite.apiRoot());
   }
 
+  public static String inventoryConfig() {return String.format("%s/inventory/config", ApiTestSuite.apiRoot());}
```

Comment:
Why are these two paths separate? Is there another endpoint provided under `/inventory/config`?